### PR TITLE
User registration

### PR DESCRIPTION
**Pull request by nmonterroso**
@Wikia/services-team 

Adding user-registration service from `:tools:create-new-service:run`. 

Also adding `ConsulBundle` and the service's `Module` to the generated `Application`

### CODE REVIEW

File: service/user-registration/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# UserRegistration
+Please enter a short, general description about what this service does
```
Comment (ArturKlajnerok) [inline service/user-registration/README.md]: Can you add some basic information to readme ?

**Comment by ArturKlajnerok** [comment]
Other than one small comment, LGTM :+1:

File: service/user-registration/README.md (hunk 1 above)
Comment (nmonterroso) [inline service/user-registration/README.md]: d79831f

**Comment by drsnyder** [comment]
:+1:

**Pull request merged**